.github/ISSUE_TEMPLATE/suggest-a-new-rule.md

```diff
@@ -0,0 +1,20 @@
+---
+name: New rule suggestion
```

Comment:
This template should have a pointer to https://github.com/dotnet/roslyn-analyzers/blob/master/GuidelinesForNewRules.md

Reply:
We also need to update that guidelines doc to add that if the new suggestion is about usage of a .NET API, it should instead be filed at https://github.com/dotnet/runtime/issues with [code-analyzer](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Acode-analyzer) label.

Reply:
Sure!

Reply:
Done